src/webots/Makefile

```diff
@@ -180,11 +180,9 @@ endif
 
 ifeq ($(MAKECMDGOALS),release)
 CFLAGS += -O3 -DNDEBUG
-ifeq ($(OSTYPE),darwin)
-STRIP = strip $(TARGET)
-else
-LD_FLAGS += -s
-endif
+ ifneq ($(OSTYPE),darwin)
+  LD_FLAGS += -s
+ endif
```

Comment:
We usually don't indent if/endif in Makefile

Reply:
Ok, I just homogenized the indentation, by dropping it in the entire file.